graph-key/src/app.js

```diff
@@ -0,0 +1,113 @@
+import {
+  createSvgCanvas,
+  createSvgPath,
+  createSvgCircle
+} from "./create-elements-utils";
+
+import {
+  translateHtmlToSvgPoint,
+  getCoordinatesOfSvgCircle,
+  generateSvgPathAttribute
+} from "./utils";
+
+import "../style.css";
+import "../graph-key.css";
+
+const config = {
+  el: document.querySelector(".graph-key"),
+  pointsCount: 4,
+  radius: 10,
+  size: 300,
+  padding: 30
+};
```

Comment:
I really like what I see here, didn't see anything wrong so far! What are your plans for this?

What's missing? :)

Reply:
I don't have any plans, just wanted to make this))

What do you mean about missing?)

Reply:
Perhaps when you do the first combination it saves it and then only one second combination it turns green or red if it matches the first. And a reset button to start over